client/coral-framework/components/Slot.js

```diff
@@ -4,10 +4,10 @@ import styles from './Slot.css';
 import {connect} from 'react-redux';
 import {getSlotElements} from 'coral-framework/helpers/plugins';
 
-function Slot ({fill, inline = false, plugin_config: {debug, ...config}, ...rest}) {
+function Slot ({fill, inline = false, plugin_config: {...config}, ...rest}) {
   return (
-    <div className={cn({[styles.inline]: inline, [styles.debug]: debug})}>
-      {getSlotElements(fill, {...config, ...rest})}
+    <div className={cn({[styles.inline]: inline, [styles.debug]: config.debug})}>
+      {getSlotElements(fill, {...rest, ...config})}
```

Comment:
Now `plugin_config` can essentially overwrite any props. I do believe you mean sth. like this
```js
function Slot ({fill, inline = false, plugin_config: config, ...rest}) {
  return (
    <div className={cn({[styles.inline]: inline, [styles.debug]: config.debug})}>
      {getSlotElements(fill, {...rest, config})}
    </div>
  );
}
```